definition/alertmanager.go

```diff
@@ -257,21 +281,8 @@ func (c *PostableApiAlertingConfig) UnmarshalYAML(unmarshal func(interface{}) er
 func (c *PostableApiAlertingConfig) Validate() error {
 	receivers := make(map[string]struct{}, len(c.Receivers))
 
-	var hasGrafReceivers, hasAMReceivers bool
```

Comment:
This should be safe now because in Grafana we have full control over the integrations and it is not possible to define upstream integration in the current APIs